core/jazz_splunk-kinesis-log-streamer/index.js

```diff
@@ -36,8 +36,9 @@ function handler(event, context, callback) {
   logger.debug('Received event:' + JSON.stringify(event));
   let errorHandler = errorHandlerModule();
   let config = configData.getConfig(event, context);
+  let isSplunkEnabled = (global_config.ENABLE_SPLUNK && (global_config.ENABLE_SPLUNK).toLowerCase() == 'true') ? true : false;
```

Comment:
To handle boolean & string cases, I would recommend this - 
```suggestion
  let isSplunkEnabled = (global_config.ENABLE_SPLUNK && (global_config.ENABLE_SPLUNK).toString().toLowerCase() === 'true') ? true : false;
```

Reply:
updated